releng/publish-doc

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+BRANCH=gh-pages
+TEMPDIR=temp
+git clone -b $BRANCH --single-branch git@github.com:viatra/massif.git $TEMPDIR
+cd $TEMPDIR
+git rm -r *
```

Comment:
If you delete everything, then it will remove the theme file and make deployment fail (see 474dac3af5e7e9f297088f323c65c518af611f32 )

Reply:
Deployment succeeds because we are serving plain HTML files, not a Jekyll-built site: https://help.github.com/articles/using-a-static-site-generator-other-than-jekyll/